src/lib/Zikula/Bundle/CoreInstallerBundle/Command/InstallCommand.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Copyright Zikula Foundation 2014 - Zikula CoreInstaller bundle.
+ *
+ * This work is contributed to the Zikula Foundation under one or more
+ * Contributor Agreements and licensed to You under the following license:
+ *
+ * @license GNU/LGPLv3 (or at your option, any later version).
+ * @package Zikula
+ *
+ * Please see the NOTICE file distributed with this source code for further
+ * information regarding copyright and licensing.
+ */
+
+namespace Zikula\Bundle\CoreInstallerBundle\Command;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Output\Output;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\ArrayInput;
+
+class InstallCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setDescription('Install Zikula from the command line.')
+            ->setHelp(<<<EOT
+<info>zikula:install</info> command installs zikula.
+EOT
+            )
+            ->setName('zikula:install');
+    }
+
+    protected function execute(InputInterface $xinput, OutputInterface $output)
+    {
+        define('_ZINSTALLVER', \Zikula_Core::VERSION_NUM);
+        $commandNames = array(
+            0 => "zikula:install:getdata",
+            1 => "zikula:install:createtables",
+            2 => "zikula:install:finish",
```

Comment:
yes. was only temporarily left in the PR hoping that I could figure out a way to make the whole command work, but that seems less and less likely. In order to do so, there would need to be a way to reload the entire container, but from what I can see this is impossible.